lib/import-export-visitor.js

```diff
@@ -717,6 +717,13 @@ function processRemoval(removal) {
   }
 }
 
+function safeKey(key) {
+  if (/^\d+$|^[_$a-zA-Z]\w*$/.test(key)) {
```

Comment:
That's what I was saying. I don't think it is in this case so it can be removed.